icons/folder_type_server.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_server</title><path d="M27.5,5.5H18.2L16.1,9.7H4.4V26.5H29.6V5.5Zm0,4.2H19.3l1.1-2.1h7.1Z" style="fill:#53c1b6"/><path d="M21.283,10.952V12.27a.623.623,0,0,1,.141,0l8.6,1.719a.721.721,0,0,1-.141,1.429.767.767,0,0,1-.143-.015l-8.453-1.69v.8a.7.7,0,0,1,.116,0l8.6,1.4a.722.722,0,0,1-.116,1.434.739.739,0,0,1-.117-.009l-8.48-1.382v.636a.628.628,0,0,1,.1-.007l8.6,1.25a.721.721,0,0,1-.1,1.435.79.79,0,0,1-.1-.007l-8.492-1.235v.727a.613.613,0,0,1,.085-.008l8.6,1.01a.722.722,0,0,1-.084,1.438c-.028,0-.055,0-.084,0l-8.514-1v10H30.9V12.876Zm8.013,12.5a.6.6,0,1,1,.6-.6A.6.6,0,0,1,29.3,23.456Z" style="fill:#4a9ad1"/><path d="M13.607,21.13c.175-.009.351-.015.525-.015,2.268,0,4.709.773,4.709,2.47v5.972a1.624,1.624,0,0,1-.129.632H21.3V10.952l-7.7,3.848Z" style="fill:#4a9ad1"/><path d="M14.1,31.069c2.07,0,3.75-.656,3.75-1.466V28.536a3.952,3.952,0,0,1-.578.287l-.059.024a8.84,8.84,0,0,1-3.113.509,8.816,8.816,0,0,1-3.113-.509l-.059-.024a3.952,3.952,0,0,1-.578-.287V29.6C10.35,30.413,12.03,31.069,14.1,31.069Z" style="fill:#4a9ad1"/><path d="M11.3,28.444a5.876,5.876,0,0,0,.979.279,9.82,9.82,0,0,0,3.642,0,5.877,5.877,0,0,0,.979-.279c.7-.274.95-.578.95-.753V26.415a4.03,4.03,0,0,1-.637.311,8.841,8.841,0,0,1-3.113.505,8.841,8.841,0,0,1-3.113-.507,4.076,4.076,0,0,1-.637-.311v1.277C10.35,27.866,10.6,28.169,11.3,28.444Z" style="fill:#4a9ad1"/><path d="M11.3,26.32a7.964,7.964,0,0,0,2.8.449,7.964,7.964,0,0,0,2.8-.449c.7-.274.95-.578.95-.753V24.229a3.9,3.9,0,0,1-.784.4,8.391,8.391,0,0,1-2.966.49,8.416,8.416,0,0,1-2.966-.489,3.869,3.869,0,0,1-.784-.406v1.336C10.35,25.743,10.6,26.046,11.3,26.32Z" style="fill:#4a9ad1"/><ellipse cx="14.1" cy="23.096" rx="3.75" ry="1.466" style="fill:#4a9ad1"/></svg>
```

Comment:
@robertohuertasm Question: Is your screenshot the thing before or after the change?